server/AccManagement.java

```diff
@@ -0,0 +1,50 @@
+package newbank.server;
+
+public class AccManagement{
+
+    //showAccounts - prints out account names and balances
```

Comment:
Since you named the function pretty descriptively, I'd say the comment here is unnecessary